pints/tests/test_abc_smc.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+#
+# Tests the basic methods of the adaptive covariance MCMC routine.
```

Comment:
```suggestion
# Unit tests for the SMC ABC class.
```

Reply:
This (and a few others) are commitable suggestions, btw, you can use the commit button above (or "add to batch") to make the suggested changes directly :D